src/test/scala/nl.knaw.dans.easy.bagstore/server/BagsServletSpec.scala

```diff
@@ -260,7 +260,7 @@ class BagsServletSpec extends TestSupportFixture
 
       val expectedFilePath = store1.resolve("01/000000000000000000000000000001/bag-revision-1/data/y")
 
-      header("Content-Length").toLong shouldBe Files.size(expectedFilePath)
+      response.header("Content-Length").toLong shouldBe Files.size(expectedFilePath)
       Source.fromInputStream(response.inputStream).mkString shouldBe Source.fromFile(expectedFilePath.toFile).mkString
```

Comment:
I'm getting warnings here and below. Please use `better.files.File` instead:

```suggestion
      Source.fromInputStream(response.inputStream).mkString shouldBe File(expectedFilePath).contentAsString
```